server/services/GameLogic.js

```diff
@@ -112,10 +112,30 @@ class GameLogic {
             this.setStatus(statusTypes.GAME_OVER);
             socket.emitStatus(this.roomId, this.status);
             const winner = this.players.reduce((prev, current) => (prev.score > current.score) ? prev : current);
-            socket.emitWinner(this.roomId, { username: winner.username });
+            const topscore = winner.score;
+            console.log(topscore)
+            console.log(this.players)
+            const drawers = this.drawers(topscore)
```

Comment:
Also you don't really need the `topscore` variable, can just do `const drawers = this.drawers(winner.score)`